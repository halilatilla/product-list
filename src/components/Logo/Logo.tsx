import { FC } from 'react'
import classnames from 'classnames'
import Link from 'next/link'
import Image from 'next/image'

import ILogo from './Logo.types'

const Logo: FC<ILogo> = ({ className, ...rest }) => {
  return (
    <Link href="/">
      <a className={classnames(className)} {...rest}>
        <Image src="/assets/logo.svg" alt="logo" width={212} height={36} />
      </a>
    </Link>
  )
}

export default Logo
