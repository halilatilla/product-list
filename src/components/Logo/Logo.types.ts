export default interface ILogo {
  className?: string
}
