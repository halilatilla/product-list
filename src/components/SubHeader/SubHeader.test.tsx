import { render } from '@src/test/test-utils'

import Logo from './SubHeader'

describe('Logo', () => {
  it('renders correctly', () => {
    const { container } = render(<Logo />)
    expect(container).toMatchSnapshot()
  })
})
