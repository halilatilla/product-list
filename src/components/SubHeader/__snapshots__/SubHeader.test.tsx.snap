// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div>
  <div
    class="subHeader flex-center-between"
  >
    <div>
      <h1
        class="title"
      >
        cep telefonları 
      </h1>
      
    </div>
    <select
      class="select"
    >
      <option
        value=""
      >
        sıralama
      </option>
      <option
        value="min"
      >
        en düşük fiyat
      </option>
      <option
        value="max"
      >
        en yüksek fiyat
      </option>
      <option
        value="az"
      >
        en yeniler (A-Z)
      </option>
      <option
        value="za"
      >
        en yeniler (Z-A)
      </option>
    </select>
  </div>
</div>
`;
